.github/workflows/release-update-stable-tag.yml

```diff
@@ -0,0 +1,192 @@
+name: 'Release: Update stable tag'
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version (in X.Y.Z or X.Y format)'
+        required: true
+        default: ''
+      dry-run:
+        description: 'Do a dry-run of the stable tag update (if UNCHECKED, it will commit the change!)'
+        type: boolean
+        required: false
+        default: true
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  validate-release-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.extract-version.outputs.version }}
+      branch: ${{ steps.extract-version.outputs.branch }}
+    steps:
+      - name: Validate release version
+        id: extract-version
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const versionInput = '${{ github.event.inputs.version }}';
+
+            // Match X.Y.Z or X.Y formats
+            const versionMatch = versionInput.match(/^(\d+\.\d+)(?:\.(\d+))?$/);
+
+            if (versionMatch) {
+              const major = versionMatch[1]; // X.Y
+              const patch = versionMatch[2] || '0'; // Z (default to 0 if not provided)
+
+              const version = `${major}.${patch}`;
+              const branch = `release/${major}`;
+
+              console.log(`Input version: ${versionInput}`);
+              console.log(`Normalized version: ${version}`);
+              console.log(`Release branch: ${branch}`);
+              core.setOutput('version', version);
+              core.setOutput('branch', branch);
+            } else {
+              core.setFailed(`Version ${versionInput} does not match the expected format (X.Y.Z or X.Y)`);
+              process.exit(1);
+            }
+
+  verify-release-branch:
```

Comment:
Given this job has the previous one as a prerequisite and it's only a couple of steps, maybe we can just merge both jobs and perform all validation in one?

I also think validation should include checking that:
- The version we're trying to set exists as a tag in SVN (`svn list <SVN_URL>/tags/<TAG>` can be used for this).
- The stable tag in `trunk`'s readme.txt is different from the version we're trying to set as stable (`svn cat` can be useful to fetch the readme from `trunk`).
  - Under normal circumstances, the new version should be _higher_ than the current one except if we're doing a revert, in which case I think we should have a checkbox to explicitly allow reverts, so that we don't mistakenly set the new stable tag to an old release.

Reply:
Good idea, I've added those two checks as well.